app/helpers/host_description_helper.rb

```diff
@@ -0,0 +1,141 @@
+module HostDescriptionHelper
+  UI.register_host_description do
+    multiple_actions_provider :base_multiple_actions
+    overview_fields_provider :base_status_overview_fields
+    overview_fields_provider :base_host_overview_fields
+    overview_buttons_provider :base_host_overview_buttons
+    title_actions_provider :base_host_title_actions
+  end
+
+  def base_multiple_actions
+    actions = []
+    if authorized_for(:controller => :hosts, :action => :edit)
+      actions.concat [
+        { :action => [_('Change Group'), select_multiple_hostgroup_hosts_path], :priority => 100 },
```

Comment:
'Change group' and 'Build hosts' swapped places, was that intentional?

![multiple-actions-before](https://user-images.githubusercontent.com/2664090/56125157-8b579d80-5f78-11e9-8fd3-3fc8f88c48bb.png)


![multiple-actions-after](https://user-images.githubusercontent.com/2664090/56125159-8db9f780-5f78-11e9-8600-40974a90b35e.png)